### ISSUE TITLE
@throws tags with namespaces broken

### ISSUE THREAD

**Issue by rcrowe** [squizlabs/PHP_CodeSniffer#322]
Like #222, I think the rule should be relaxed to take into account namespaces. With both of the examples below, an error is raised complaining about a missing @throws tag:

``` php
use Exception;

/**
 * @throws \Exception
 */
protected function registerStatsd()
{
    throw new Exception;
```

``` php
/**
 * @throws Exception
 */
protected function registerStatsd()
{
    throw new \Exception;
```

**Comment by jrfnl** [comment]
I believe this has been fixed via PR #2322, which will be included in the 3.4.1 release.

**Comment by gsherwood** [comment]
I can confirm this is fixed in 3.4.1
